packages/nimble-components/src/menu-button/tests/menu-button.stories.ts

```diff
@@ -49,27 +51,73 @@ const metadata: Meta<MenuButtonArgs> = {
         }
     },
     argTypes: {
+        label: {
+            name: 'default',
+            description:
+                'The text content of the button. This will be hidden when `content-hidden` is set but should always be provided; see [Accessibility](./?path=/docs/components-menu-button--docs#accessibility) for more info.',
```

Comment:
I dunno if you see something different but in Firefox and Chrome clicking this link isn't working for me. It updates the URL but does not scroll. If I refresh the page it scrolls to the right spot.

Maybe for same page references we can just say See `Accessibility` section below?

```suggestion
                'The text content of the button. This will be hidden when `content-hidden` is set but should always be provided; see the `Accessibility` section for more info.',
```

Reply:
I was seeing the same thing but had thought it was maybe just a macOS bug. Changed to match your suggestion.